python/modules/IcePy/Util.h

```diff
@@ -8,20 +8,6 @@
 #include "Config.h"
 #include "Ice/Ice.h"
 
-//
-// These macros replace Py_RETURN_FALSE and Py_RETURN_TRUE. We use these
-// instead of the standard ones in order to avoid GCC warnings about
-// strict aliasing and type punning.
-//
-#define PyRETURN_FALSE return incFalse()
-#define PyRETURN_TRUE return incTrue()
-
-#define PyRETURN_BOOL(b)                                                                                               \
-    if (b)                                                                                                             \
-        PyRETURN_TRUE;                                                                                                 \
-    else                                                                                                               \
-        PyRETURN_FALSE
-
 namespace IcePy
 {
     inline PyObject* incRef(PyObject* obj)
```

Comment:
I am thinking of removing this one too. Using the  Py_XINCREF directly makes the code more readable. Any preferences?

Reply:
Yes, I would use Py_XINCREF directly.